docs/installing.md

```diff
@@ -1,19 +1,19 @@
 # Installing
 
-Download the latest [DevilutionX release](https://github.com/diasurgical/devilutionX/releases), or [build from source](./docs/building.md), and extract the contents to a location of your choosing.
+Download the latest [DevilutionX release](https://github.com/diasurgical/devilutionX/releases), or [build from source](./docs/building.md), and extract the contents to a location of your choosing or [build from source](./docs/building.md).
```

Comment:
building is already in the middle of the sentience.

Reply:
derp

Reply:
Should it be at the end? Because you don't extract the contents really if you build it.